tests/acceptance/bootstrap/FeatureContext.php

```diff
@@ -2317,6 +2317,14 @@ public function substituteInLineCodes(
 					"getUUIDv4Regex"
 				],
 				"parameter" => []
+			],
+			[
+				"code" => "%file_uuidv4_pattern%",
+				"function" => [
+					__NAMESPACE__ . '\TestHelpers\GraphHelper',
+					"getUUIDv4Regex"
+				],
+				"parameter" => []
```

Comment:
won't `file_id_pattern` regex work here or its different?

Reply:
We have:
```
[
				"code" => "%file_id_pattern%",
				"function" => [
					__NAMESPACE__ . '\TestHelpers\GraphHelper',
					"getFileIdRegex"
				],
				"parameter" => []
			],
```
for `file_id_pattern` and the method `getFileIdRegex` used above returns `\$` as per:
```
public static function getFileIdRegex(): string {
		return self::getUUIDv4Regex() . '\\\$' . self::getUUIDv4Regex() . '!' . self::getUUIDv4Regex();
	}
```

but we only needed `$`.
So I think `file_id_pattern` won't work.